Calculator/Calculator/CalculatorItem.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  CalculatorItem.swift
+//  Calculator
+//
+//  Created by 박종화 on 2023/05/30.
+//
+
+import Foundation
+
+protocol CalculatorItem {
+    
+}
```

Comment:
```suggestion
protocol CalculatorItem {}
```

내부 구현이 없이 빈 경우엔 본문의 중괄호도 붙이는 것이 일반적인 컨벤션 같습니다.

Reply:
빈 경우엔 중괄호 붙이기! 수정했습니다!